pkg/settings/setting.go

```diff
@@ -92,6 +92,7 @@ var (
 	PartnerChartDefaultBranch         = NewSetting("partner-chart-default-branch", "main")
 	FleetDefaultWorkspaceName         = NewSetting("fleet-default-workspace-name", "fleet-default") // fleetWorkspaceName to assign to clusters with none
 	ShellImage                        = NewSetting("shell-image", "rancher/shell:v0.1.6")
+	HelmPostDeletionImage             = NewSetting("helm-post-deletion-image", "rancher/shell:v0.1.6") // need to update the value in chart/values.yaml for any change in this line
```

Comment:
@cbron this is to address your [comment](https://github.com/rancher/rancher/pull/31670/files#r595497658):
> 1. could work but it needs to load the entire rancher codebase for it to work, so that would take good amount of time.
> 2. If someone forgets then airgap won't work for this, much safer to add to requirements struct and also a comment there. That seems like best solution for now.

Reply:
Need to run gofmt on this line. Also I think we should add here and say to update this line: https://github.com/rancher/rancher/blob/master/pkg/image/resolve.go#L248
That resolve.go line needs to always be in sync with values.yaml for this hook, that is our failsafe if someone changes this setting and not the values.yaml.